src/main/java/duke/WriteToFile.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+import java.io.FileWriter;   // Import the FileWriter class
+import java.io.IOException;  // Import the IOException class to handle errors
+import java.util.ArrayList;
+
+
+public class WriteToFile {
+    public static void main(ArrayList taskList) {
+        try {
+            FileWriter myWriter = new FileWriter("D:\\Documents\\Modules\\Sem 03\\CS2113\\iP\\data\\duke.txt");
```

Comment:
Might not work on other systems with no D drive.
This won't work in non-Windows machine because of different path naming convention.
You may consider using relative path instead of absolute path.